src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Vaccination/HomeVaccinationTableViewCell.swift

```diff
@@ -0,0 +1,61 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class HomeVaccinationTableViewCell: UITableViewCell {
+	
+	// MARK: - Overrides
+
+	override func awakeFromNib() {
+		super.awakeFromNib()
+
+		certificateTitle.text = AppStrings.HealthCertificate.Info.Home.title
+		certificateBody.text = AppStrings.HealthCertificate.Info.Home.body
+		inProgressLabel.text = AppStrings.HealthCertificate.Info.Home.inProgress
+	}
+	
+	override func setHighlighted(_ highlighted: Bool, animated: Bool) {
+		super.setHighlighted(highlighted, animated: animated)
+
+		containerView.setHighlighted(highlighted, animated: animated)
+	}
+	// MARK: - Internal
+
+	func configure(with cellModel: HomeVaccinationCellModel) {
+		guard !isConfigured else { return }
+
+		cellModel.$vaccinatedPersonName.assign(to: \.text, on: nameLabel).store(in: &subscriptions)
+
+		cellModel.$isVerified.sink { [weak self] isVerified in
+			DispatchQueue.main.async {
+				if isVerified {
+					self?.containerView.backgroundColor = .enaColor(for: .buttonPrimary)
+					self?.inProgressLabel.isHidden = true
+					self?.iconView.image = UIImage(named: "Vaccination_full")
+				} else {
+					self?.containerView.backgroundColor = .enaColor(for: .riskNeutral)
+					self?.inProgressLabel.isHidden = false
+					self?.iconView.image = UIImage(named: "Vacc_Incomplete")
+				}
+			}
+			
+		}.store(in: &subscriptions)
+		
+		isConfigured = true
+	}
```

Comment:
```suggestion
		cellModel.$isVerified
			.receive(on: DispatchQueue.main.ocombine)
			.sink { [weak self] isVerified in
				if isVerified {
					self?.containerView.backgroundColor = .enaColor(for: .buttonPrimary)
					self?.inProgressLabel.isHidden = true
					self?.iconView.image = UIImage(named: "Vaccination_full")
				} else {
					self?.containerView.backgroundColor = .enaColor(for: .riskNeutral)
					self?.inProgressLabel.isHidden = false
					self?.iconView.image = UIImage(named: "Vacc_Incomplete")
				}
			}
			.store(in: &subscriptions)
		
		isConfigured = true
	}
```

You can directly receive on the main queue.
Additional suggestion: make the three properties publishers on the view model and assign them all separately here.